logs.js

```diff
@@ -62,62 +62,95 @@ try {
       .setTimestamp();
        logChannel.send({embeds: [embed]})
       await member.roles.add(member.guild.roles.cache.find(role => role.name === 'Pilots'))
+      await addUserToFCP(member);
+
   } catch (error) {
           sendToSentry(error, "Guild member add log")
   }
 }
 
+// 286 - Add member to TPC Portal - JP
+// Seperate function to catch this error seperate from the 
+// logging funtion above.
+async function addUserToFCP(member)
+{
+  try{
+    // 286 - Add member to the TPC Portal - JP
+    const url = `${config.fcpBaseUrl()}/api/users/new`;
+    const response = await fetch(url, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json',
+            'Authorization': `Bearer ${process.env.FCP_TOKEN}`
+            },
+            body: JSON.stringify(
+              {
+                'id': `'${member.id}'`
+              }
+            )
+    });
+    if (response.status != 200) {

```

Comment:
This needs to check for a 201 response. Also, need to have !== not != as this will cause unintended consequences.